src/CmsmsClient.php

```diff
@@ -33,18 +29,17 @@ public function __construct(GuzzleClient $client, $productToken)
 
     /**
      * @param CmsmsMessage $message
+     * @param string $recipient
      * @throws CouldNotSendNotification
      */
-    public function send(CmsmsMessage $message)
+    public function send(CmsmsMessage $message, $recipient)
     {
-        if (empty($message->originator)) {
+        if (is_null(Arr::get($message->toXmlArray(), 'FROM'))) {
             $message->setOriginator(config('services.cmsms.originator'));
```

Comment:
Should be `$message->originator(config('services.cmsms.originator'));` due to the removed `set` prefixes in the message class.